services/streammanager/kinesis/kinesismanager.go

```diff
@@ -44,6 +44,9 @@ func NewProducer(destination *backendconfig.DestinationT, o common.Opts) (*Kines
 	if err != nil {
 		return nil, err
 	}
+	// Given we use this for only streaming, we can set a high number of idle connections (default is 2)
+	sessionConfig.MaxIdleConnsPerHost = 50
```

Comment:
expressvpn has more than 50 kinesis destinations, so it is still not enough for them. I see two options:

- Set this value to be equal to the number of destinations
- Make it configurable, so that we can tune it for different customers.